### ISSUE TITLE
VOID macro declaimed as inline 

### ISSUE THREAD

**Issue by slyrus** [vseloved/rutils#59]
The following line in misc.lisp declaims the macro VOID to be inline:
```
(declaim (inline or2 and2 xor2 void true))
```
This should probably be removed.

**Comment by jcguu95** [comment]
Indeed! According to the [standard (Declaration INLINE)](https://www.lispworks.com/documentation/HyperSpec/Body/d_inline.htm#inline), inlining macros has no effect (nor an error), unless a compiler macro is provided for the name.

> [inline](https://www.lispworks.com/documentation/HyperSpec/Body/d_inline.htm#inline) and [notinline](https://www.lispworks.com/documentation/HyperSpec/Body/d_inline.htm#notinline) declarations otherwise have no effect when the lexically visible definition of function-name is a [macro](https://www.lispworks.com/documentation/HyperSpec/Body/26_glo_m.htm#macro) definition.
